setup.py

```diff
@@ -21,5 +21,12 @@
     ],
     python_requires='>=3.8',
     package_dir={'':'.'},
-    install_requires=['pyzmq', 'msgpack']
-)
\ No newline at end of file
+    install_requires=['pyzmq', 'msgpack'],
+    extras_require={
+        'test': [
+            'pytest',
+            'pytest-asyncio',
+            'pyjwt'
+        ]
+    }
+)
```

Comment:
These are "must be" installs, which is not true for zero. It is only needed for the test and not needed for package distribution. Please remove this.